certificate-policy.md

```diff
@@ -168,42 +169,44 @@ The key words "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMME
-
-The Certification Practice Statement shall state the CA's practice on processing CAA Records for Fully Qualified Domain Names. The CA shall log all actions taken, if any, consistent with its processing practice.
-
-The FPKIPA and/or a designee and/or the CAs shall publish test Web pages that allow Application Software Suppliers to test their software with Subscriber Certificates from the Issuing CAs that chain up to the publicly trusted Root Certificate. At a minimum, separate Web pages shall be published showing Subscriber Certificates that are (i) valid, (ii) revoked, and (iii) expired.
-
+The FPKIPA shall publicly post this CP on \<INSERT URL>, ensuring it is readily accessible on a 24x7 basis.
+
+Each CA shall disclose the following information through a publicly accessible Repository: 
+- CA Certificates
+- Certificate Revocation Lists (CRLs)
+- Online Certificate Status Protocol responses
+- CPS documents
+- Subscriber Agreements
+- Audit Reports
+
+Web pages that allow for testing certificate validation up to the U.S. Federal Public Trust TLS Root CA can be found at:
+• https://valid. \<2nd LEVEL DOMAIN TBD>
+• https://revoked. \<2nd LEVEL DOMAIN TBD>
+• https://expired. \<2nd LEVEL DOMAIN TBD>
+
+ 
 ## 2.3 Time or frequency of publication
-The FPKIPA and CAs shall update and publish the Certificate Policy and Certification Practice Statements in accordance with Section 2.0.
+The FPKIPA and CAs shall update and publish the CP and CPS documents within thirty (30) days after being approved.
 
-All CAs approved to issue a CA certificate shall post to the Repository any issued CA certificate as soon as possible after issuance but no later than 15 days after issuance.  The FPKIPA or designee shall disclose the CA certificate and submit the CA certificate to trust stores and applicable databases, such as the Common CA Database, within thirty (30) days of issuance.
+Root CA(s) shall post to the Repository any issued CA Certificate as soon as possible after issuance but no later than fifteen (15) days after issuance.  The FPKIPA or designee shall disclose and submit the CA Certificate, CPS, and Audit Report(s) to trust store operators and applicable databases, such as the Common CA Database, as required by the trust store operator policies.
 
 Each CA shall publish CRLs in accordance with Section 4.9.7.
 
 ## 2.4 Access controls on repositories
-Each CA shall make its Repository publicly available in a read-only manner.
+Each CA shall make its Repository publicly available in a read-only manner.  Repository information shall be protected by unauthorized modification.
```

Comment:
Grammar typo here: should be `protected from unauthorized`, not `protected by unauthorized`.

But also, why do we need this?